ntoskrnl/mm/ARM3/expool.c

```diff
@@ -1465,11 +1465,45 @@ ExpExpandBigPageTable(
     ASSERT(KeGetCurrentIrql() == DISPATCH_LEVEL);
 
     /* Make sure we don't overflow */
-    if (!NT_SUCCESS(RtlULongMult(2,
-                                 OldSize * sizeof(POOL_TRACKER_BIG_PAGES),
-                                 &NewSizeInBytes)))
+    if (!Shrink)
```

Comment:
Maybe just change the name to `Expand`? Or swap the cases

Reply:
Alright.